comp/core/secrets/secretsimpl/secrets.go

```diff
@@ -661,4 +684,9 @@ func (r *secretResolver) GetDebugInfo(w io.Writer) {
 	if err != nil {
 		fmt.Fprintf(w, "error rendering secret info: %s", err)
 	}
+
+	if r.refreshIntervalScatter {
+		fmt.Fprintf(w, "The first secret refresh will happen at a random time between the starting of the agent and the set refresh interval")
```

Comment:
Let give more precise information to users. It might also be useful if we need to troubleshoot this feature (tracking the time of the first refresh will be useful).
```suggestion
		fmt.Fprintf(w, "'secret_refresh interval' enabled: the first refresh will happen at '<insert time here>'' and then every %s", <first refresh>, r.refreshInterval)
```

Reply:
That makes sense. I've added the randomized scatter duration to the resolver struct, so that it can be picked up in this method.